docs/panresponder.md

```diff
@@ -93,15 +93,9 @@ const App = () => {
   const panResponder = useRef(
     PanResponder.create({
       onMoveShouldSetPanResponder: () => true,
-      onPanResponderGrant: () => {
-        pan.setOffset({
-          x: pan.x._value,
-          y: pan.y._value,
-        });
-      },
       onPanResponderMove: Animated.event([null, {dx: pan.x, dy: pan.y}]),
       onPanResponderRelease: () => {
-        pan.flattenOffset();
+        pan.extractOffset();
```

Comment:
This has the same effect as before without relying on private member variables from Animated.value() like the previous example did.